setup.cfg

```diff
@@ -28,6 +28,7 @@ install_requires =
     cirq-core~=0.14.1
     cirq-google~=0.14.1
     orquestra-quantum
+    openfermion>=1.5
```

Comment:
I think it should be ~=, as probably the next `openfermion` update will be incompatible with `cirq-core~=0.14` as it has been recently bumped to `1.0.0`